src/vorta/views/repo_add_dialog.py

```diff
@@ -211,6 +214,11 @@ def password_listener(self):
             self.passwordLabel.setText(translate('utils', msg))
             return not bool(msg)
 
+    def cancel_job(self):
+        if self.job:
+            self.job.cancel()
```

Comment:
Sadly this only cancelt the job if its already running. If the job manager hasn't picked up the job yet, that is the job still sits in the queue, it won't be cancelled.

Reply:
Hi! I'm a bit confused testing this, any suggestion on how can I test this ?

Reply:
Currently there is no way to reproduce this with the GUI. So you can decide not to deal with it but instead write a code comment on the issue.

Reply:
> Currently there is no way to reproduce this with the GUI. So you can decide not to deal with it but instead write a code comment on the issue.

The code will be included in `cancel_job` method since it will check for both, if running or in queue, should I create a duplicate method for that and comment it ?

Reply:
Hi! @real-yfprojects , Since this is not reproducible, I'm not sure that the code will work or not, then is it okay to push the code even it is commented ?

Reply:
I can only decide after seeing the code.